components/ui-api-layer/internal/authn/oidc.go

```diff
@@ -0,0 +1,38 @@
+package authn
+
+import (
+	"k8s.io/apiserver/pkg/authentication/authenticator"
+	"k8s.io/apiserver/pkg/authentication/request/bearertoken"
+	"k8s.io/apiserver/plugin/pkg/authenticator/token/oidc"
+)
+
+// OIDCConfig represents configuration used for JWT request authentication
+type OIDCConfig struct {
+	IssuerURL            string
```

Comment:
Is this used?

Reply:
It does here: https://github.com/kyma-project/kyma/pull/2317/files#diff-9b1753afc0ab91ae785587efe44043fdR24

Reply:
Yes, but I don't see this value to be set anywhere. So it's always empty string?

Reply:
It is loaded from env variables here: https://github.com/kyma-project/kyma/pull/2317/files#diff-b2f3b478fbdef32eaac69d47d9e1fa24R84